tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/documentation/ErrorsTest.java

```diff
@@ -52,17 +52,20 @@ public void setup() throws IOException {
 
     @Test(description = "Test error type")
     public void testErrorAsAType() {
-        Assert.assertEquals(testModule.errors.size(), 5, "Six errors expected");
```

Comment:
This test covers all the behaviors we have mentioned in the issue?

Reply:
Yes